docs/UPDATE_CORE.md

```diff
@@ -64,10 +64,10 @@ But be aware: there might be **migrations** that are applied to your accounts da
 
 If you already have a core git checkout, you can skip the first step.
 
-1. clone the core repo, right next to your desktop repo folder: `git clone git@github.com:deltachat/deltachat-core-rust.git`
+1. clone the core repo, right next to your desktop repo folder: `git clone git@github.com:chatmail/core.git deltachat-core-rust`
 2. go into your core checkout and run `git pull` to update it to the newest version, then create a branch for your changes
 3. run `python3 deltachat-rpc-server/npm-package/scripts/make_local_dev_version.py`
-4. run `npm i` and `npm run build` inside `../deltachat-core-rust/deltachat-jsonrpc/typescript/`
+4. run `npm i` and `npm run build` inside `../core/deltachat-jsonrpc/typescript/`
```

Comment:
```suggestion
4. run `npm i` and `npm run build` inside `../deltachat-core-rust/deltachat-jsonrpc/typescript/`
```

Reply:
This would mean we have to explain to new developers to check out the core using
`git clone https://github.com/chatmail/core deltachat-core-rust`
which is not quite intuitive

Imo its better to let the current developers to get used to the new name, than let new developers "learn" the legacy name

Reply:
Well yeah, just "core" is real weird, I bet people would still want to name the directory properly. So yeah, I'd vote for keeping `deltachat-core-rust`.

Reply:
So now the script supports both and you can set CORE_REPO_CHECKOUT as env var.
The main reason was, that other deltachat developers also use the link_core script and for them it might be annoying to figure out, why it's not working anymore.